submissions/kuslosolodka/html-css-popup/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Popup</title>
+    <link rel="stylesheet" href="styles/style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="popup-button">
+      <label for="popup-btn">
+        <img class="popup-icon"
+            src="assets/icons/popup-button.png"
+            alt="popup icon">
+          </label>
+    </div>
+    <input type="checkbox" id="popup-btn" />
+    <div class="popup-menu">
+      <div class="icons">
+        <a href="#" tabindex="0"
+          ><img
+            src="assets/icons/calendar.png"
+            alt="calendar icon"
+          />Calendar</a
+        >
+        <a href="#" tabindex="0"
+          ><img src="assets/icons/mail.png" alt="mail icon" />Mail</a
+        >
+        <a href="#" tabindex="0"
+          ><img src="assets/icons/news.png" alt="news icon" />News</a
+        >
+        <a href="#" tabindex="0"
+          ><img src="assets/icons/maps.png" alt="calendar icon" />Map</a
+        >
+        <a href="#" tabindex="0"
+          ><img src="assets/icons/files.png" alt="calendar icon" />Files</a
+        >
+        <a href="#" tabindex="0"
+          ><img
+            src="assets/icons/contacts.png"
+            alt="calendar icon"
+          />Contacts</a
+        >
+        <a href="#" tabindex="0"
+          ><img
+            src="assets/icons/tanslation.png"
+            alt="calendar icon"
+          />Translate</a
+        >
+        <a href="#" tabindex="0"
+          ><img src="assets/icons/videos.png" alt="calendar icon" />Videos</a
+        >
+        <a href="#" tabindex="0"
+          ><img src="assets/icons/search.png" alt="calendar icon" />Search</a
+        >
+        <a href="#" tabindex="0"
+          ><img src="assets/icons/photos.png" alt="calendar icon" />Photo</a
+        >
+        <a href="#" tabindex="0"
+          ><img src="assets/icons/apps.png" alt="calendar icon" />Apps</a
+        >
+        <a href="#" tabindex="0"
+          ><img
+            src="assets/icons/documents.png"
+            alt="calendar icon"
+          />Document</a
+        >
+      </div>
+      <div class="button-field">
+        <label class="load-more" for="load-more-btn">
+          <p>More</p>
+        </label>
+      </div>
+      <input type="checkbox" id="load-more-btn" />
+      <div class="hidden-icons" tabindex="1">
+        <a href="#" tabindex="0"
+          ><img src="assets/icons/chat.png" alt="chat icon" />Chat</a
+        >
+        <a href="#" tabindex="0"
```

Comment:
`tabindex=0` on interactive elements makes no sense. [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) about it